homeassistant/components/insteon_hub.py

```diff
@@ -36,20 +36,19 @@ def setup(hass, config):
     _LOGGER.warning("Component disabled at request from Insteon. "
                     "For more information: https://goo.gl/zLJaic")
     return False
-    # pylint: disable=unreachable
-    import insteon
+    # import insteon
 
-    username = config[DOMAIN][CONF_USERNAME]
-    password = config[DOMAIN][CONF_PASSWORD]
-    api_key = config[DOMAIN][CONF_API_KEY]
+    # username = config[DOMAIN][CONF_USERNAME]
+    # password = config[DOMAIN][CONF_PASSWORD]
+    # api_key = config[DOMAIN][CONF_API_KEY]
 
-    global INSTEON
-    INSTEON = insteon.Insteon(username, password, api_key)
+    # global INSTEON
+    # INSTEON = insteon.Insteon(username, password, api_key)
 
-    if INSTEON is None:
-        _LOGGER.error("Could not connect to Insteon service")
-        return False
+    # if INSTEON is None:
+    #    _LOGGER.error("Could not connect to Insteon service")
+    #    return False
 
-    discovery.load_platform(hass, 'light', DOMAIN, {}, config)
+    # discovery.load_platform(hass, 'light', DOMAIN, {}, config)
 
-    return True
+    # return True
```

Comment:
I would prefer if we just don't touch this file. If anything, just add a comment to disable the Quantified Code warning.